pkg/cloudevents/transport/pubsub/internal/connection.go

```diff
@@ -49,6 +51,15 @@ const (
 	DefaultRetentionDuration = 25 * time.Hour
 )
 
+var DefaultReceiveSettings = pubsub.ReceiveSettings{
+	// Pubsub default receive settings will fill in other values.
+	// https://godoc.org/cloud.google.com/go/pubsub#Client.Subscription
+
+	// Override the default number of goroutines.
+	NumGoroutines: 1000,
```

Comment:
maybe add a comment of why 1k. Say it's a magic number, but has shown performance improvements empirically, and such...

Reply:
Done